keyboards/sleepy_craft_studios/sleepy_keeb_split/keyboard.json

```
@@ -0,0 +1,100 @@
+{
+    "manufacturer": "Sleepy Craft Studios",
+    "keyboard_name": "sleepy_keeb_split",
+    "maintainer": "Sleepy Craft Studios",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["B5", "B6", "B2", "B3", "B1", "F7"],
+        "rows": ["C6", "D7", "E6", "B4"]
+    },
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 18,
+        "split_count": [9, 9]
+    },
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "D2",
+        "usb_detect": {
+            "enabled": true
+        }
+    },
+    "url": "https://sleepycraftstudios.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0002",
+        "vid": "0x7373"
+    },
+    "ws2812": {
+        "pin": "F4"
+    },
+    "layouts": {
+        "LAYOUT": {
```

This looks to be a 4x12 ortho layout? 

```suggestion
    "community_layouts": [ "ortho_4x12" ],
    "layouts": {
        "LAYOUT_ortho_4x12": {
```

---

I tried that and it wouldn't compile. Did it compile for you with these changes @drashna ?

---

you'd need to update all of the keymaps when doing this, too. If you didn't do that, then yeah, it would generate errors.

---

Ah, I must have missed one. Thanks!